temporalcli/commands.gen.go

```diff
@@ -0,0 +1,785 @@
+// Code generated. DO NOT EDIT.
```

Comment:
Do we need to check this in? Can it not be autogenerated as part of the Go build?

Reply:
No, you cannot auto-generate anything in Go build really (can't auto-run any code on build). It's the Go way to do it this way (proto, mocks, this type of thing, etc). This is great for code readers in GH of course.